packages/destination-actions/src/destinations/stackadapt-audiences/forwardProfile/functions.ts

```diff
@@ -1,8 +1,52 @@
 import { RequestClient } from '@segment/actions-core'
 import { Payload } from './generated-types'
 import { domain } from '..'
+import { createHash } from 'crypto'
+
+const EXTERNAL_PROVIDER = 'segmentio'
+
+const standardFields = new Set([
+  'email',
+  'firstName',
+  'lastName',
+  'phone',
+  'emailMarketingStatus',
```

Comment:
we'll use `marketingStatus` in the source pod to do some parsing on this field. This is to align with the naming in some shared lib, https://github.com/StackAdapt/sa-go-sharelib/blob/285979c004d144900ecc16960fe8b9d349c85c60/cdp/cdp-gateway/gen/server.pb.go#L214